luaui/Widgets/gui_game_type_info.lua

```diff
@@ -108,16 +108,17 @@ end
 
 local sec = 0
 function widget:Update(dt)
+	if messages[2] == nil then return end
```

Comment:
Instead of this runtime check, you could also just disable `Update` right in `Initialize`:
```lua
if unba then
  messages[2] = bla
else
  widgetHandler:RemoveCallIn("Update") -- only handles blink for the unba message
end
```